src/pages/workspaces/workspace/analysis/modals/DeleteDiskChoices.ts

```diff
@@ -0,0 +1,199 @@
+import { Fragment } from 'react'
+import { code, div, h, p, span } from 'react-hyperscript-helpers'
+import TitleBar from 'src/components/TitleBar'
+import { cloudServices } from 'src/data/gce-machines'
+import { isDataprocConfig, isGceConfig, isGceWithPdConfig, RuntimeConfig } from 'src/libs/ajax/leonardo/models/runtime-config-models'
+import * as Utils from 'src/libs/utils'
+import { computeStyles } from 'src/pages/workspaces/workspace/analysis/modals/modalStyles'
+import { WarningTitle } from 'src/pages/workspaces/workspace/analysis/modals/WarningTitle'
+import { RadioBlock, SaveFilesHelp, SaveFilesHelpAzure, SaveFilesHelpRStudio } from 'src/pages/workspaces/workspace/analysis/runtime-common-components'
+import { getPersistentDiskCostMonthly } from 'src/pages/workspaces/workspace/analysis/utils/cost-utils'
+
+
+export const DeleteDiskChoices = ({
+  persistentDiskCostDisplay,
+  deleteDiskSelected,
+  setDeleteDiskSelected,
+  toolLabel,
+  cloudService
+}:
+    {
+      persistentDiskCostDisplay: String
+      deleteDiskSelected:boolean
+      setDeleteDiskSelected:(p1:boolean)=>void
+      toolLabel:string
+      cloudService: any
+    }) => {
+  const getCurrentMountDirectory = () => {
+    const rstudioMountPoint = '/home/rstudio'
```

Comment:
seems like its worth having a `type MountPoint = '/home/rstudio' | '/home/jupyter'` and have function `getMountDir  = (toolLabel: ToolLabel): MountPoint` for type safety. Also, `toolLabel` should not be compared to magic string `RStudio`, rather compared to `toolLabels.RStudio.label (not correct syntax here, but point should be clear)`